src/api/Folder.js

```diff
@@ -90,6 +104,64 @@ class Folder extends Item {
         return cache.has(this.key);
     }
 
+    /**
+     * Takes representations fields from items in collection and assigns thumbnailUrls
+     * to items, with the use of the callback
+     *
+     * @param {Collection} collection - collection with items that have representations field populated
+     * @param {Token} token - access_token
+     * @param {(item: BoxItem, index: number, thumbnailUrl: string) => void} callback - Function to call with generated thumbnailUrl
+     * @return {void}
+     */
+    formatRepresentations(
+        collection: Collection,
```

Comment:
could destructure the items out of the collection since we're not using collection anywhere